utils.py

```diff
@@ -148,6 +148,23 @@ def write_report(
         create_report_sheet("ELSTER - Summary", df_summary, writer)
 
 
+def get_daily_rate(daily_rates: pd.DataFrame, date: datetime, currency: str):
+    if date in daily_rates[currency]:
+        return daily_rates[currency][date]
+    else:
+        # On weekends the currency exchange doesn't operate. Go back some days in time to find a valid value
```

Comment:
wouldn't it make sense to go ahead, i.e. find the next valid day after the weekend?

Reply:
So far I was under the impression that the previous day applies (since it's already known). However, after some research I found [this article](https://www.interactivebrokers.com/en/education/currency-settlement-holidays.php) which explains that it indeed goes further ahead in time.
I'll fix it, thanks for noticing.

Reply:
I would say this is more intuitive :D You can always defer the settlement of a transaction into the future, but pushing it into the past usually seems counterintuitive to me.